vocal/Script_VOCAL_unified.R

```diff
@@ -470,9 +483,8 @@ compute_alert_levels_v1 <- function(pheno_table_wide) {
 var_pheno_summary_wide_with_alert = compute_alert_levels_v1(var_pheno_summary_wide)
 
 prediction_overview = var_pheno_summary_wide_with_alert %>% group_by(alert_level) %>% count()
-message("\n Prediction Results")
-print(prediction_overview)
-message("\n")
+log_info("Prediction Results:")
+log_info(prediction_overview)
```

Comment:
You could also be closer to the log_* version if you just directly check the log_threshold():

```suggestion
if (log_threshold() >= TRACE) {
print(prediction_overview)
}
```